docs/UserGuide.md

```diff
@@ -9,6 +9,28 @@ NetworkBook is a **desktop contact book application**. You can use it to network
 
 ### <u>Category 1 - Add contact information</u>
 
+### Remove duplicate contacts on adding contact
```

Comment:
Please move this under the `create` command. Pull the latest from `branch-update-ug` and resolve conflicts.